multi-int-store/src/test/java/com/connexta/multiintstore/MultiIntStoreIntegrationTest.java

```diff
@@ -31,6 +41,22 @@ public void setup() throws Exception {
   @Test
   public void testContextLoads() {}
 
+  @Test
+  public void handleMetadataCallback() throws Exception {
+    mockMvc.perform(MockMvcRequestBuilders.post("/store/1234")
+            .contentType("application/json")
+            .content("{\n" + "    \"id\": \"1234\",\n" + "    \"status\": \"COMPLETE\",\n"
+                    + "    \"type\": \"ddms5.0\",\n" + "    \"mimeType\": \"application/xml\",\n"
+                    + "    \"bytes\": \"256\",\n"
+                    + "    \"location\": \"https://localhost:8080\",\n" + "    \"security\": {\n"
+                    + "        \"classification\": \"U\",\n"
+                    + "        \"ownerProducer\": \"ownerProducer\"\n" + "    }\n" + "}"))
+            .andExpect(status().isOk());
+    List<Metadata> metadatas = new ArrayList<>();
+    repository.findAll().forEach(metadatas::add);
+    assertThat(metadatas.size(), is(1));
```

Comment:
✏️Nit: Seems like there is an easier way to verify the size than copying to a new List, e.g. `assertThat(Iterables.size(repository.findAll()), is(1));` or a hamcrest matcher.

Reply:
Once we actually get real metadata to store we'll find it by the real ID, but for now I think this _should_ be fine.